sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotHardcodeCredentials_DefaultValues.cs

```diff
@@ -63,6 +63,16 @@ public void StandardAPI(SecureString secureString, string nonHardcodedPassword,
             new PasswordDeriveBytes("hardcoded", byteArray, "strHashName", 1); // Noncompliant
             new PasswordDeriveBytes("hardcoded", byteArray, "strHashName", 1, cspParams); // Noncompliant
         }
+
+        public void CompliantParameterUse(string pwd)
+        {
+            string query1 = "password=?";
+            string query2 = "password=:password";
+            string query3 = "password=:param";
+            string query4 = "password='"+pwd+"'";
```

Comment:
The way this is handled produces FN in similar cases. We should add test to document the FN and explain why.
`"password=hardcoded;user='" + login + "'"`

Reply:
No really related to this commit but indeed since only `LiteralExpressionSyntax` nodes are covered, we should document this FN with an unit test.

Reply:
Clean as you code :)